poc/ts/src/BecomingArguments.ts

```
@@ -0,0 +1,155 @@
+      return this.container.getNamed(paramType, namedValue);
+    } else {
+      // Otherwise, use the parameter type
+      return this.container.get(paramType);
+    }
+  }
+
+  /**
+   * Validates that all constructor parameters have explicit decorator declarations
+   */
+  private validateParameterDecorators(
+    constructor: Function,
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)